source/opt/propagator.h

```diff
@@ -187,10 +187,14 @@ class SSAPropagator {
   SSAPropagator(ir::IRContext* context, const VisitFunction& visit_fn)
       : ctx_(context), visit_fn_(visit_fn) {}
 
-  // Run the propagator on function |fn|. Returns true if changes were made to
+  // Runs the propagator on function |fn|. Returns true if changes were made to
   // the function. Otherwise, it returns false.
   bool Run(ir::Function* fn);
 
+  // Returns true if the |i|th argument for |phi| comes through a CFG edge that
+  // has been marked executable.
```

Comment:
MIght be worth saying the numbering for |i| is 0-based and includes result id and type id ??
Or just say you're going to use it with GetSingleWordOperand